.github/workflows/build-packages.yml

```diff
@@ -42,6 +45,9 @@ jobs:
          mkdir -p jans/jans-src/opt/
          cp -rp packaging/${{ matrix.build_files }}/* jans/
          wget https://raw.githubusercontent.com/JanssenProject/jans/main/jans-linux-setup/jans_setup/install.py -O jans/install.py
+         sudo add-apt-repository ppa:deadsnakes/ppa
+         sudo apt-get update
+         sudo apt-get install -y python${{ matrix.python_version }}
```

Comment:
Use matrix based form with python3 step instead of apt-get 

```
    - name: Set up Python 3.7
      uses: actions/setup-python@v3.0.0
      with:
        python-version: ${{ matrix.python_version }}
```

Reply:
@moabu not able to make install.py work with action/setup-python.
1st it installs python under /opt which works only for current user, when running install.py with sudo it can't find python. I fixed this by updating root PATH, after getting issue with install.py not funding some other python packages that ldap3 etc.